### DISCUSSION TITLE
issues with running a malloc profile

### DISCUSSION THREAD

**Discussion by georgew5656** [async-profiler/async-profiler#970] [Q&A]
i'm trying to test a very basic setup on a centos instance (no containers)where i have a java application that runs in a loop and allocates and frees direct memory. i want to use the malloc tracking option on async-profiler to get stack traces for the calls.

i have not really tried running asprof as an agent, right now i'm running via the cli
./asprof start -e malloc <pid>
./asprof stop <pid>

this seems to work fine when i do a -e alloc or omit it entirely, but when i try tracking malloc's what seems to happen is the java process hits the malloc call breakpoint and gets stuck there, the ./asprof call also gets stuck and never tracks anything.

is there something v obvious i am doing wrong here?

**Comment by apangin** [comment]
Thank you for your interest in async-profiler.
To understand an issue you're observing and to provide a quality troubleshooting advise, we need more information about the problem.
Please submit a new [bug report](https://github.com/async-profiler/async-profiler/issues/new?template=bug-report.yml), providing as much context as possible, ideally, filling in all fields.
